internal/telemetry/middleware_test.go

```diff
@@ -25,10 +24,6 @@ func newTestMux() http.Handler {
 
 func Test_HTTPMetricsHandler(t *testing.T) {
 
-	chain := middleware.NewChain()
```

Comment:
Middleware chain can be replaced with long form (otherwise I was getting a cyclical import error)